<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiRoom</class>
 <widget class="QWidget" name="MultiRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="HostButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton { background-color: rgb(170, 85, 0); border-radius: 3px; color: rgb(255, 255, 255); }
QRadioButton:hover { background-color: rgb(0, 170, 255); }</string>
   </property>
   <property name="text">
    <string>Play as host</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ClientButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton { background-color: rgb(170, 85, 0); border-radius: 3px; color: rgb(255, 255, 255); }
QRadioButton:hover { background-color: rgb(0, 170, 255); }</string>
   </property>
   <property name="text">
    <string>Play as client</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditIP">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditPort">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBuild">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>300</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(170, 85, 0); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(0, 170, 255); } QPushButton:disabled { background-color: rgb(170, 170, 170); }</string>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConnect">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>260</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(170, 85, 0); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(0, 170, 255); } QPushButton:disabled { background-color: rgb(170, 170, 170); }</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStart">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>350</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(170, 85, 0); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(0, 170, 255); } QPushButton:disabled { background-color: rgb(170, 170, 170); }</string>
   </property>
   <property name="text">
    <string>Start!</string>
   </property>
  </widget>
  <widget class="QLabel" name="IPPort">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIP">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>IP Address:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBackToMenu">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(170, 85, 0); border-radius: 3px; color: rgb(255, 255, 255); } QPushButton:hover { background-color: rgb(0, 170, 255); }</string>
   </property>
   <property name="text">
    <string>BackToMenu</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
